feat(curriculum): dice game project (#50819)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/64d0ad7f530d6d024785484a.md

---
id: 64d0ad7f530d6d024785484a
title: Step 74
challengeType: 0
dashedName: step-74
---

# --description--

> To count the occurences of each number in an array, use a `for` loop. Within the loop, create a variable named `num` to represent the current element of the `arr` array.

Reviewer: The instruction can be improved.

```suggestion
To count the occurences of each number, use a `for` loop to iterate through every element of the array `arr`. Within the loop, create a variable named `num` to represent the current element of the `arr` array.
```
